examples/resources/awscc_lambda_function/lambda_basic.tf

```diff
@@ -15,29 +15,42 @@ resource "awscc_iam_role" "main" {
   })
 }
 
-data "archive_file" "main" {
+data "archive_file" "example" {
   type        = "zip"
-  source_file = "main.py"
+  source_file = "index.py"
   output_path = "lambda_function_payload.zip"
 }
 
-resource "awscc_lambda_function" "main" {
-  function_name = "lambda_function_name"
+resource "awscc_s3_bucket" "lambda_assets" {
+}
+
+resource "aws_s3_object" "zip" {
+  source = data.archive_file.example.output_path
+  bucket = awscc_s3_bucket.lambda_assets.id
+  key    = "index.zip"
+}
+
+resource "awscc_lambda_function" "example" {
+  function_name = "example"
   description   = "AWS Lambda function"
   code = {
-    zip_file = data.archive_file.main.output_path
+    s3_bucket = awscc_s3_bucket.lambda_assets.id
+    s3_key    = aws_s3_object.zip.key
   }
   package_type  = "Zip"
-  handler       = "main.lambda_handler"
+  handler       = "index.handler"
   runtime       = "python3.10"
   timeout       = "300"
   memory_size   = "128"
-  role          = awscc_iam_role.main.arn
+  role          = awscc_iam_role.example.arn
   architectures = ["arm64"]
   environment = {
     variables = {
       MY_KEY_1 = "MY_VALUE_1"
       MY_KEY_2 = "MY_VALUE_2"
     }
   }
-}
\ No newline at end of file
+  ephemeral_storage = {
+    size = 10240 # Min 512 MB and the Max 10240 MB
```

Comment:
Can we use the minimum or a lower value here to minimize costs to run the example?

Reply:
Done.